met-web/src/components/engagement/form/EngagementForm.tsx

```diff
@@ -79,7 +77,6 @@ const EngagementForm = () => {
 
         setEngagementFormError({
```

Comment:
I am assuming we can drop this call all together since the description doesn't affect the formerror anymore